index.js

```diff
@@ -95,6 +95,39 @@ class Conf {
 		this.store = plainObject();
 	}
 
+	static isEqual(newValue, oldValue) {
```

Comment:
This makes it unnecessarily available on the public interface to consumers. This could just be a private helper method in the file, similar to `plainObject` to prevent unintentional usage.

Reply:
If refactored into a shared method, extraction of the equal logic may be unnecessary.